src/app/docs/description-list/description-list-docs.component.html

```diff
@@ -0,0 +1,90 @@
+<sky-docs-demo-page
+  moduleName="SkyDescriptionListModule"
+  moduleSourceCodePath="src/app/public/modules/description-list/"
+  pageTitle="Description list"
+>
+  <sky-docs-demo-page-summary>
+    Description lists display lists of terms and pairs each term with a description.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo
+    heading="Name-value pair demo"
```

Comment:
```suggestion
    heading="Standard mode demo"
```
Not sure if referencing the mode is the best option here, but I think so. Definitely don't want `Name-value` since we abandoned that terminology, and "Standard term-description pair demo" felt a bit long-winded.